include/grpc++/support/channel_arguments.h

```diff
@@ -64,7 +64,9 @@ class ChannelArguments {
   /// Set the compression algorithm for the channel.
   void SetCompressionAlgorithm(grpc_compression_algorithm algorithm);
 
-  /// Set the GRPCLB fallback timeout (in ms) for the channel.
+  /// Set the grpclb fallback timeout (in ms) for the channel. Here, fallback 
```

Comment:
What does the timeout do?  A user reading this might think it is (e.g.) the amount of time that we use the fallback addresses before giving up and waiting for the balancer response.